### ISSUE TITLE
Layout/CommentIndentation flags commented out else/elsif branches

### ISSUE THREAD

**Issue by SvcDptDev** [rubocop-hq/rubocop#7143]
When I comment out an `else`/`elsif` branch from an `if` conditional, the Layout/CommentIndentation Cop flags the `else`/`elsif` line.

I'm not 100% sure this is really a bug. There is a strong argument against committing commented out code. My personal use case here is a code branch that I only want to use when I'm debugging. I could easily do something other than comment the branch out, but the Layout/CommentIndentation cop flagging it feels like a bug.

This is really edge case though, so if I'm wrong about it being a bug or if it would difficult to fix, feel free to close.

--------

## Expected behavior

```ruby
if 1 < 2
  puts 'hello'
# elsif 1 > 2
  # puts 'world'
else
  puts 'goodbye'
end
```

I expect no RuboCop offenses.

## Actual behavior

```
Offenses:

test.rb:3:1: C: Layout/CommentIndentation: Incorrect indentation detected (column 0 instead of 2).
# elsif 1 > 2
^^^^^^^^^^^^^
```

## Steps to reproduce the problem

1. Create a `test.rb` file.
2. Paste in the example from above.
3. Run `rubocop test.rb`.

## RuboCop version

```
0.71.0 (using Parser 2.6.2.0, running on ruby 2.6.1 x86_64-linux)
```

**Comment by buehmann** [comment]
To be honest, your commented code also looks strange to me. I would use the following:
```ruby
if 1 < 2
  puts 'hello'
# elsif 1 > 2
#   puts 'world'
else
  puts 'goodbye'
end
```

**Comment by SvcDptDev** [comment]
Hmm. I use Sublime Text and after checking, I see that it comments the same way you just showed, and that that doesn't generate a violation. I'm not sure how I came up with the example code I posted. Thanks for pointing out my mistake.
